demo/app.js

```diff
@@ -502,10 +523,38 @@ const Demo = React.createClass({
     });
   },
 
+  _nextSibling () {
+    console.log('next');
+  },
+
+  _previousSibling () {
+    console.log('previous');
+  },
+
+  _toggleDrawer () {
+    this.setState({
+      showDrawer: !this.state.showDrawer
+    });
+  },
+
   render () {
+    const navContent = {
+      duration: 200,
+      onNextClick: this._nextSibling,
```

Comment:
I would rename `_nextSibling` to something like `_clickNextSibling` to indicate that it is a function.